adapters/nft-key/index.js

```diff
@@ -0,0 +1,164 @@
+require("dotenv").config();
+
+const moment = require("moment");
+const BigNumber = require("bignumber.js");
+const Ethereum = require("../../sdk/EVMC");
+const axios = require("axios");
+const URL = "http://nft-sales-service.dappradar.com/open-source";
+const KEY = process.env.DAPPRADAR_API_KEY;
+const path = require("path");
+
+class NFT_K {
+    // stands for NFT Key
+    constructor() {
+        this.name = "NFTK";
+        this.symbol = null;
+        this.token = "0x0000000000000000000000000000000000000000";
```

Comment:
For native AVAX token use `avax` instead of `0x0`.

Reply:
Was a bit confused on what was being asked here, but can update this as well, and then I assumed for all chains would be the same, ie ftm, bnb, matic, etc.